app/views/pages/governance/index.njk

```diff
@@ -1,56 +1,61 @@
 {% set pageTitle = "Governance" %}
-{% set pageDescription = "Technical governance ensures we’re doing the right things and, doing them correctly." %}
+{% set pageDescription = "Technical governance ensures we’re doing the right things and, doing them correctly.
+          Our various boards are each responsible for their own part in this:" %}
 
 {% extends 'includes/layout.njk' %}
 
 {% block outerBody %}
-<section class="nhsuk-hero">
-  <div class="nhsuk-width-container nhsuk-hero--border">
+<nav class="nhsuk-breadcrumb{% if params.classes %} {{ params.classes }}{% endif %}" aria-label="Breadcrumb"
+{%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+<div class="nhsuk-width-container">
+  <ol class="nhsuk-breadcrumb__list">
+    {%- for item in params.items %}
+      {%- if item.href %}
+        <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" href="/"{% for attribute, value in item.attributes %} {{attribute}}="{{value}}"{% endfor %}>Home</a>{% if not loop.last %}{% endif %}</li>
+      {%- endif -%}
+    {% endfor %}
+    <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" href="/"{% for attribute, value in item.attributes %} {{attribute}}="{{value}}"{% endfor %}>Home</a></li>
+    <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" {% for attribute, value in item.attributes %} {{attribute}}="{{value}}"{% endfor %}>{{ pageTitle }}</a></li>
+  </ol>
+</div>
+</nav>
+{% endblock %}
+
+
+{% block body %}
     <div class="nhsuk-grid-row">
-      <div class="nhsuk-grid-column-two-thirds">
-        <div class="nhsuk-hero__wrapper">
-          <h1 class="nhsuk-u-margin-bottom-3">{{ pageTitle }}</h1>
-          <p class="nhsuk-body-l nhsuk-u-margin-bottom-0">{{ pageDescription }}</p>
+      <div class="nhsuk-grid-column-full app-component-reading-width">
```

Comment:
No styles exist for `app-component-reading-width` so I don't think this is class is needed